### PR TITLE
Merge v1.0-beta to master 

### PR DESCRIPTION
**Pull request by pradeepnschrodinger**
## Description

Merge latest beta to master. This was a clean merge without conflicts.

## Motivation and Context

Fixes [SS-28116](https://jira.schrodinger.com/browse/SS-28116)

## How Has This Been Tested?

Used local FDT site. Verified that:
* the site contains all the new examples (and that they work just fine)
* the docs are all upto date (roadmap, codebase, api)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by wcjordan** [comment]
Hey @pradeepnschrodinger, I'm just getting back from vacation to look at this.  Is there anything you think needs specific review here?  Or do you just want a rubber stamp saying we're ready to make the beta branch the new master.

File: package.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
 {
   "name": "fixed-data-table-2",
-  "version": "0.8.28",
+  "version": "1.0.0-beta.30",
```
Comment (wcjordan) [inline package.json]: Do we want to switch this to not be beta anymore?

1.0.1?

Comment (pradeepnschrodinger) [reply package.json]: sounds good.

Btw why not `v1.0.0` instead of `v1.0.1` ?

Comment (wcjordan) [reply package.json]: I was thinking of the betas as v1.0.0.  But looking at react they go from 15.4.0-rc.4 to 15.4.0, so I'm good w/ the release being 1.0.0.

**Pull request merged**